stubs/xmltodict/xmltodict.pyi

```diff
@@ -0,0 +1,33 @@
+from _typeshed import SupportsRead, SupportsWrite
+from collections import OrderedDict
+from typing import Any, Mapping, overload
```

Comment:
Nit for future reference: We prefer to import `Mapping` (and other ABCs) from `collections.abc` now rather than `typing`.